frontend/app_flowy/lib/workspace/application/doc/share_bloc.dart

```diff
@@ -33,8 +37,37 @@ class DocShareBloc extends Bloc<DocShareEvent, DocShareState> {
   ExportData _convertDeltaToMarkdown(ExportData value) {
     final result = deltaToMarkdown(value.data);
     value.data = result;
+    writeFile(result);
     return value;
   }
+
+  Future<Directory> appFlowyDocumentDirectory() async {
```

Comment:
The appFlowyDocumentDirectory function is located in rust_sdk.dart file, you don't have to repeat the code.